test/runtime-utils/src/lib.rs

```diff
@@ -91,35 +92,47 @@ use frame_system::Config as SystemConfig;
 use frame_system::Pallet as System;
 use std::marker::PhantomData;
 
-pub trait RollTo {
-	fn with(_: u64);
-}
+pub trait RollTo<R: SystemConfig> {
+	type Pallet;
 
-pub struct Trivial;
+	fn with(_: R::BlockNumber);
 
-impl RollTo for Trivial {
-	fn with(_: u64) {}
+	fn roll_to(n: R::BlockNumber) {
+		let mut now = <System<R>>::block_number();
+		while now < n {
+			now += One::one();
+			Self::with(now);
+		}
+	}
 }
 
-pub struct WithWorkerHook<P>(PhantomData<P>);
+pub struct Trivial<P>(PhantomData<P>);
 
-impl<Pallet: OffchainWorker<u64>> RollTo for WithWorkerHook<Pallet> {
-	fn with(i: u64) {
-		Pallet::offchain_worker(i);
+impl<Pallet, R: SystemConfig> RollTo<R> for Trivial<(Pallet, R)>
```

Comment:
Doesn't matter much, but is there a reason to prefer `Trivial<(Pallet, R)>` over `Trivial<Pallet, R>`?